crates/swc_css_lints/src/rules/font_family_no_duplicate_names.rs

```diff
@@ -56,18 +56,19 @@ impl FontFamilyNoDuplicateNames {
                         (fonts, Some((value.to_string(), *span)))
                     }
                 }
-                ComponentValue::Str(Str {
-                    raw: Some(raw),
-                    span,
-                    ..
-                }) => {
-                    fonts.push((FontNameKind::from(raw), *span));
+                ComponentValue::Str(value) if value.raw.is_some() => {
```

Comment:
Please use patter matching instead of an if guard

Reply:
fixed